docker-compose-mastodon-update.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+git stash
+git pull
+git stash pop
+docker-compose build
+docker-compose up -d
+docker-compose run --rm web rails assets:precompile
+docker-compose run --rm web rails db:migrate
```

Comment:
I've always wondered: is it advisable to run `rails db:migrate` while the app is running? Or should we bring down the server first to let the migrations happen atomically?